packages/actor-function-factory-expression-function-coalesce/test/coalesce01-spec-test.ts

```diff
@@ -1,6 +1,8 @@
+import { ActorFunctionFactoryTermFunctionDivision } from '@comunica/actor-function-factory-term-function-division';
 import { int, decimal } from '@comunica/expression-evaluator/test/util/Aliases';
 import { Notation } from '@comunica/expression-evaluator/test/util/TestTable';
-import { runFuncTestTable } from '../util';
+import { runFuncTestTable } from '../../bus-function-factory/test/util';
```

Comment:
I guess something like this will also work here (and possible other places):
```suggestion
import { runFuncTestTable } from '@comnunica/bus-function-factory/test/util';
```